lizrd/scripts/grid.py

```diff
@@ -29,27 +29,25 @@
 from lizrd.support.code_copying import copy_code
 from lizrd.support.misc import load_with_inheritance
 
-if __name__ == "__main__":
-    parser = argparse.ArgumentParser()
-    parser.add_argument("--config_path", type=str)
-    parser.add_argument("--git_branch", type=str, default="")
-    parser.add_argument("--neptune_key", type=str, default=None)
-    parser.add_argument("--wandb_key", type=str, default=None)
-    args = parser.parse_args()
-    CLUSTER_NAME = get_machine_backend()
-    PROCESS_CALL_FUNCTION = lambda args, env: subprocess.run(
-        [str(arg) for arg in args if arg is not None], env=env
-    )
 
-    if args.config_path.endswith(".yaml"):
-        configs, all_config_paths = load_with_inheritance(args.config_path)
+def create_subprocess_args(
+    config_path,
+    git_branch,
+    neptune_key,
+    wandb_key,
+    CLUSTER_NAME,
+    skip_confirmation=False,
+    skip_copy_code=False,
+):
+    if config_path.endswith(".yaml"):
+        configs, all_config_paths = load_with_inheritance(config_path)
     else:
         raise ValueError("config path point to a .yaml")
 
     for config in configs:
-        config["params"]["git_branch"] = args.git_branch
-        config["params"]["path_to_entry_config"] = args.config_path
-        config["params"]["all_config_paths"] = ",".join(all_config_paths)
+        config["params"]["git_branch"] = git_branch
+        config["params"]["path_to_entry_config"] = config_path
+        config["params"]["all_config_paths"] = ",".join(sorted(list(all_config_paths)))
```

Comment:
all_config_paths have nondeterministic order

Reply:
ja bym te logikę sortowania wywalił do funkcji `load_with_inheritance` bo to tam robimy seta. Taka opcja sprawia że mamy rozbitą logikę tworzenia `all_config_paths` na dwie różne funkcje.